src/components/Chips/Chips.tsx

```diff
@@ -129,86 +129,96 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
       [onClick]
     );
 
+    const overrideClassName = cx(styles.chips, "chips--wrapper", className, {
+      [styles.disabled]: disabled,
+      [styles.withClose]: hasCloseButton,
+      [styles.noAnimation]: noAnimation,
+      [styles.withUserSelect]: allowTextSelection
+    });
+
+    const clickableProps = useClickable(
+      {
+        onClick: onClickCallback,
+        onMouseDown,
+        className: overrideClassName,
+        enableTextSelection: allowTextSelection,
+        disabled,
+        id,
+        dataTestId: overrideDataTestId,
+        ariaLabel: ariaLabel || label,
+        ariaHidden: false,
+        ariaHasPopup: false,
+        ariaExpanded: false,
+        style: backgroundColorStyle
+      },
+      mergedRef
+    );
+    const wrapperProps = hasClickableWrapper
```

Comment:
doesn't look too good maybe, but at least it's more straightforward which props will wrapper have

Reply:
maybe this will be more clear?
```suggestion
    const clickableWrapperProps = hasClickableWrapper
```

Reply:
Not sure, cause it's more generic than clickableWrapperProps. Can be clickable props, but can be not related to clickable
I think `wrapperProps` are better here

Reply:
let's talk about it on zoom ? didnt understand really